### ISSUE TITLE
Cross column dependencies?

### ISSUE THREAD

**Issue by blooop** [theodo/ticket-dependency-graph#37]
Is there a reason that the tool works for single columns rather than the entire board?

I was thinking of modifying the code to make it work for every card in the board and wanted to check if there was a technical/other reason why it supports single lists rather than the board. 

At the moment I am working around this by putting everything in a single column, but its already causing other problems as I need to be able to move cards between columns for other reasons. 

Thanks.

**Comment by ivanosevitch** [comment]
Hi Austin,
I'm not sure about the reasons of this design choice, as it was made early in the production creation. My best guess would be:

- It was faster to develop single-column handling than multi-column.
- It was adapted to the workflow of the development team that first used this tool (for own projet): in their scrum setting, all cards "ready to develop" would be stored in a single column. Then the dependency graph would be created, and only after the cards would be moved to the next sprint backlog.

I know some team have a use case with multiple column handling, someone even attempted (however could not finish) to implement this behavior : https://github.com/theodo/ticket-dependency-graph/pull/13
Here is another tool that implements multi-column: https://github.com/DanielPopOut/graph-dependencies

What is the use case you are thinking about that would need multiple columns?

**Comment by blooop** [comment]
Hi Ivan,

Thanks for your reply.  That makes sense about the dev process.  

I came across the other tool first, but it has other problems (syncing across multiple users) and this project has a better user interface, and the output graph looks much nicer. 

The use case is I am making a long term road-map and wanted the dependency graph as a visual representation.  However, if I want to move any of those tasks into an "in progress" or "done" column, they are now missing from the graph. 

Thanks for the link to the pr.  I was thinking of something simpler where it just loops through all columns by default, no user interface or anything to select as I can see how that would be more work.

**Comment by ivanosevitch** [comment]
> I was thinking of something simpler where it just loops through all columns by default

Some of the boards we use have a _lot_ of cards and columns, so I think that's why we prefer selecting a few specific columns :)

**Comment by blooop** [comment]
Fair enough, I've only just got started with Trello so don't have that many yet.

**Comment by ivanosevitch** [comment]
Hi @blooop , a recent feature (https://github.com/theodo/ticket-dependency-graph/pull/46) by @mpierret added a filter by labels.
Does that answer some of your use cases?

**Comment by blooop** [comment]
Yes, this looks useful for my case. Thanks for remembering!

However, I don't think the label selection is working properly in my browser (chrome).  I can select the label filter dropdown and different cards appear depending on their label, but the dropdown box is empty, so I have no idea what label I am selecting. 

![image](https://user-images.githubusercontent.com/1213400/124159762-68e7c700-da93-11eb-9bab-3e09bc313e3a.png)
